benchmark/sampler/hetero_neighbor.py

```diff
@@ -0,0 +1,119 @@
+import argparse
+import ast
+import os.path as osp
+import time
+
+import torch
+import torch_geometric.transforms as T
+import torch_sparse  # noqa
+from torch_geometric.datasets import OGB_MAG
+from torch_geometric.loader.utils import get_input_nodes
```

Comment:
Can we avoid the dependency to PyG (besides for `OGB_MAG`)? I don't think it is desperately needed for benchmarking and these functions are not part of the public API and not stable.

Reply:
Done.